### ISSUE TITLE
[Bug] `Portfolio/PO/mktcap` - Error: marketCap - not enough values to unpack (expected 2, got 0)

### ISSUE THREAD

**Issue by deeleeramone** [OpenBB-finance/OpenBBTerminal#3789]
Using the example file included, `SP_500_Portfolio.xlsx`, the optimization for market cap fails. 

```
order_book_path = '/Users/danglewood/GitHub/OpenBBTerminal/openbb_terminal/miscellaneous/portfolio_examples/allocation/SP_500_Portfolio.xlsx'
order_book = pd.read_excel(order_book_path)
order_book_cols = ['Ticker', 'Asset Class', 'Sector', 'Industry', 'Country',
       'Current Invested Amount', 'Currency']

order_book = order_book[order_book_cols]

p = openbb.portfolio.po.load(symbols_file_path=order_book_path)

weights, performance = openbb.portfolio.po.mktcap(p)
```

Which gives the output of:

```

8 Failed downloads:
- CERN: No data found, symbol may be delisted
- BLL: No data found, symbol may be delisted
- NLSN: No data found, symbol may be delisted
- CTXS: No data found, symbol may be delisted
- FB: No data found, symbol may be delisted
- ANTM: No data found, symbol may be delisted
- BRK.B: No data found, symbol may be delisted
- BF.B: No data found for this date range, symbol may be delisted
Error: 'marketCap'

Error: not enough values to unpack (expected 2, got 0)
```

**Comment by diogodanielsoaresferreira** [comment]
I am having a similar issue when I run /stocks/fa/mktcap (or call openbb.stocks.fa.mktcap()), for any stock ticker.

```
Error: 'sharesOutstanding'
Error: not enough values to unpack (expected 2, got 0)
```

**Comment by diogodanielsoaresferreira** [comment]
The issue seems to be [here](https://github.com/OpenBB-finance/OpenBBTerminal/blob/07c08df84e2af99be4ee32ab276128cafb9e7986/openbb_terminal/stocks/fundamental_analysis/yahoo_finance_model.py#L299), where the key "sharesOutstanding" is not found.

I went a bit deeper and saw that, with the current yfinance version that OpenBBTerminal is using (`yfinance==0.1.74`), locally if I run `yf.Ticker("AMZN").info`,
I get
`{'regularMarketPrice': None, 'preMarketPrice': None, 'logo_url': ''}`: a dict without "sharesOutstanding".

However, with a more up-to-date current version of yfinance (0.1.93), I get a dict with "sharesOutstanding", which is what OpenBBTerminal is expecting.

**Comment by deeleeramone** [comment]
Thanks, @diogodanielsoaresferreira!, this also resolves #3796

**Comment by jmaslek** [comment]
is this resolved after #3801 ?

**Comment by diogodanielsoaresferreira** [comment]
Yes, I don't have the same problem with the new version 👍
